tests/license_check.py

```diff
@@ -26,6 +32,19 @@
     "GNU General Public License (GPL)",
     "Public Domain",
     "The Unlicense (Unlicense)",
+    "Apache License, Version 2.0",
+    "Apache License v2.0",
+    "Apache License 2.0",
+    "GNU Lesser General Public License v2 or later (LGPLv2+)",
+    "ISC",
+    "LGPL-2.1-only",
+    "GNU General Public License v3 (GPLv3)",
```

Comment:
hmm could be an issue? 
<img width="727" alt="image" src="https://github.com/AbanteAI/mentat/assets/128252497/35da4c40-0010-4726-81f5-d48ad4d34d0a">